engine/runtime/src/main/java/org/enso/interpreter/runtime/data/DisplayArrayUtils.java

```diff
@@ -0,0 +1,50 @@
+package org.enso.interpreter.runtime.data;
+
+import com.oracle.truffle.api.interop.InteropLibrary;
+import com.oracle.truffle.api.interop.InvalidArrayIndexException;
+import com.oracle.truffle.api.interop.UnsupportedMessageException;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class DisplayArrayUtils {
+
+  public static String toDisplayString(
```

Comment:
@JaroslavTulach for now I refactored the common code to this static method. I do wonder if it is really worth making this a Truffle Node.

Reply:
If the code is behind `@TruffleBoundary` then it doesn't make any sense to make it a `Node`. The code concatenates strings - e.g. it has to be behind boundary. E.g. static method is completely fine.